configuration/pih/pih-config-haiti-mentalhealth.json

```diff
@@ -21,6 +21,7 @@
     "chwApp",
     "clinicianDashboard",
     "dataExports",
+    "dbevent",
```

Comment:
we might want to hold enabling on the laptops to start (though I guess it wouldn't hurt?)

Reply:
We might just want to get rid of the component altogether.  We can disable this via a system/runtime/global property on a server-by-server basis that doesn't require new config deployment, so probably best to limit that way if we want to hold off on the laptops.

Reply:
Probably a future ticket to remove the component, once we have it deployed everywhere?